### PR TITLE
Adding GitHub release and tag automation through versioning YAML file.

### PR DESCRIPTION
**Pull request by ro4i7**
## Description

[Provide a brief description of the changes you have made and why they are necessary.]

## Changes Made
[List the changes you have made in bullet points.]

## Checklist
- [ ] I have read and followed the contributing guidelines.
- [ ] I have tested my changes thoroughly and they work as expected.
- [ ] I have added necessary tests for the changes made.
- [ ] I have updated the documentation to reflect the changes made.
- [ ] My code follows the project's coding style and standards.
- [ ] I have added appropriate commit messages and comments for my changes.

## Related Issues
[If your changes relate to any existing issues, please list them here.]

## Additional Notes
[Provide any additional notes or context that may be helpful for the reviewers.]

Feel free to modify and customize this template as needed to fit the specific needs of the GCP Scanner project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,31 +1,20 @@
 # How to Contribute
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
+We'd love to accept your patches and contributions to this project. There are just a few small guidelines you need to follow.
```
Comment (ZetaTwo) [inline CONTRIBUTING.md:3]: Unrelated change, please do not include in this PR

File: src/gcp_scanner/crawl.py [hunk 2]
```diff
@@ -1,1035 +0,0 @@
-# Copyright 2022 Google LLC
```
Comment (ZetaTwo) [inline src/gcp_scanner/crawl.py:1]: Unrelated change, please do not include in this PR

**Review by ZetaTwo — CHANGES_REQUESTED** [review]

**Comment by ro4i7** [comment]
closing because of many issues commits into 1 PR
